rspec-mode.el

```diff
@@ -546,7 +553,9 @@ to navigate to the example or method corresponding to point."
                            "^\\.\\./"))
           (relative-file-name (file-relative-name a-file-name (rspec-spec-directory a-file-name))))
       (rspec-specize-file-name (expand-file-name (replace-regexp-in-string replace-regex "" relative-file-name)
-                                                 (rspec-spec-directory a-file-name))))))
+                                                 (if rspec-unit-test-dir
```

Comment:
Couldn't you simplify all this code if the new variable had the default value `"/spec/"`?